code/game/turfs/closed/walls.dm

```diff
@@ -138,16 +144,14 @@
 
 /turf/closed/wall/attack_hulk(mob/user)
 	..()
-	if(prob(hardness))
-		playsound(src, 'sound/effects/meteorimpact.ogg', 100, TRUE)
-		user.say(pick(";RAAAAAAAARGH!", ";HNNNNNNNNNGGGGGGH!", ";GWAAAAAAAARRRHHH!", "NNNNNNNNGGGGGGGGHH!", ";AAAAAAARRRGH!" ), forced = "hulk")
-		dismantle_wall(1)
-	else
-		playsound(src, 'sound/effects/bang.ogg', 50, TRUE)
-		add_dent(WALL_DENT_HIT)
-		user.visible_message("<span class='danger'>[user] smashes \the [src]!</span>", \
-					"<span class='danger'>You smash \the [src]!</span>", \
-					"<span class='hear'>You hear a booming smash!</span>")
+	playsound(src, 'sound/effects/meteorimpact.ogg', 100, TRUE)
+	user.say(pick(";RAAAAAAAARGH!", ";HNNNNNNNNNGGGGGGH!", ";GWAAAAAAAARRRHHH!", "NNNNNNNNGGGGGGGGHH!", ";AAAAAAARRRGH!" ), forced = "hulk")
+	playsound(src, 'sound/effects/bang.ogg', 50, TRUE)
+	add_dent(WALL_DENT_HIT)
+	user.visible_message("<span class='danger'>[user] smashes \the [src]!</span>", \
+				"<span class='danger'>You smash \the [src]!</span>", \
+				"<span class='hear'>You hear a booming smash!</span>")
+	take_damage(100, BRUTE, "melee", FALSE, user.dir, 100)
```

Comment:
```suggestion
	take_damage(hulk_damage(), BRUTE, "melee", FALSE, user.dir, 100)
```

Reply:
and again, not a fan of us having 2 procs on turf and obj that basically do the same thing.

Reply:
this specific proc is really weird, all it does is return 150. might as well make it a define or not even use it

Reply:
I don't mind just make sure that both obj and Turf uses the same method and value.

Reply:
A define would be okay.